templates/zerodb/zerodb.py

```diff
@@ -122,7 +119,13 @@ def info(self):
         """
         Return disk information
         """
-        return self._zerodb_sal.info
+        info = self._zerodb_sal.info
+        try:
+            self.state.check('status', 'running', 'ok')
+            info['running'] = True
```

Comment:
this looks strange. I someone wants to know if the zerodb is running, it can just check it's state. I don't think we should start giving this information in here